src/__stories__/header-story.tsx

```diff
@@ -18,7 +18,11 @@ export default {
     },
 };
 
-export const Default: StoryComponent = () => {
+type Args = {
+    bleedValue: boolean;
+};
```

Comment:
Please don't use Args in this story.

It's fine (event preferred) to use `Args` in stories, but this story is using inline controls (`useTextField`, `useCheckbox`, etc) and it isn't a good idea to mix both (args + inline controls). I suggest you to just check the bleed checkbox in your test.

In the future we can migrate this story to use `Args` instead, but that would be a different task

Reply:
Oh, okay. I hadn't thought of it that way. I will check how to perform correctly and even today I upload the changes. Thanks for the feedback and explanation.